src/AutoRest.CSharp/LowLevel/AutoRest/LowLevelTarget.cs

```diff
@@ -48,9 +48,11 @@ public static void Execute(GeneratedCodeWorkspace project, InputNamespace inputN
             foreach (var client in library.RestClients)
             {
                 var codeWriter = new CodeWriter();
-                var lowLevelClientWriter = new LowLevelClientWriter();
-                lowLevelClientWriter.WriteClient(codeWriter, client);
+                var xmlDocWriter = new XmlDocWriter();
+                var lowLevelClientWriter = new LowLevelClientWriter(codeWriter, xmlDocWriter, client);
+                lowLevelClientWriter.WriteClient();
                 project.AddGeneratedFile($"{client.Type.Name}.cs", codeWriter.ToString());
+                project.AddGeneratedDocFile($"Docs/{client.Type.Name}.xml", xmlDocWriter.ToString());
```

Comment:
What if we regenerate, do we need to clean up the docs folder?

Reply:
> What if we regenerate, do we need to clean up the docs folder?

@lirenhe The docs folder is `Generated/Docs` which is under `Generated` folder. And `Generated` folder will be cleaned up (probably by autorest), before we write back generated codes.